sporks.json

```diff
@@ -1,950 +1,995 @@
+              "execution-state-wal-chunks": ""
+            }
+          },
+          "flow-go-tag": "v0.12.6",
+          "flow-dps-tag": "",
+          "cadence-tag": "v0.10.2",
+          "docker-tag": "v0.12.6",
+          "access-node": "access-001.mainnet3.nodes.onflow.org:9000"
+        },
+        {
+          "id": 2,
+          "name": "Mainnet-2",
+          "spork-date": "2020-11-13T16:00:00Z",
+          "root-height": "8742959",
+          "root-parent-id": "b35fdb189d21a95df7f19f941786f748d9854a8b93b1e555b51cda7d9f53a6e1",
+          "root-state-commit": "d6a25be552ed93213df0ffc2e8c7f39f6401c04cbf22bac7a4b84d3c9493f005",
+          "git-commit-hash": "4ef68efb935c0e3393ae3966752ece5e7739bab4",
+          "state-artefacts" : {
+            "gcp": {
+              "root-checkpoint-file": "https://storage.googleapis.com/flow-genesis-bootstrap/mainnet-2-execution/public-root-information/root.checkpoint",
+              "root-protocol-state-snapshot": "",
+              "node-info": "https://storage.googleapis.com/flow-genesis-bootstrap/mainnet-2-execution/public-root-information/node-infos.pub.json",
+              "execution-state-wal-chunks": ""
+            },
+            "s3": {
+              "root-checkpoint-file": "",
+              "root-protocol-state-snapshot": "",
+              "node-info": "",
+              "execution-state-wal-chunks": ""
+            }
+          },
+          "flow-go-tag": "v0.11.1",
+          "flow-dps-tag": "",
+          "cadence-tag": "v0.9.2",
+          "docker-tag": "v0.11.1",
+          "access-node": "access-001.mainnet2.nodes.onflow.org:9000"
+        },
+        {
+          "id": 1,
+          "name": "Mainnet-1",
+          "spork-date": "2020-10-13T16:00:00Z",
+          "root-height": "7601063",
+          "root-parent-id": "ab1ee18b6e1c0ee11cc021c26a17c694c627699a576e85f7013cd743bdbc7877",
+          "root-state-commit": "6e1adf15689eaf5ea6859bcdd0b510f5eb4c34dac878d8577b3f65bc20c3f312",
+          "git-commit-hash": "114d45436e7d9052e910c98a1e40f730e3fd12d7",
+          "state-artefacts" : {
+            "gcp": {
+              "root-checkpoint-file": "https://storage.googleapis.com/flow-genesis-bootstrap/mainnet-1-execution/public-root-information/root.checkpoint",
+              "root-protocol-state-snapshot": "",
+              "node-info": "https://storage.googleapis.com/flow-genesis-bootstrap/mainnet-1-execution/public-root-information/node-infos.pub.json",
+              "execution-state-wal-chunks": ""
+            },
+            "s3": {
+              "root-checkpoint-file": "",
+              "root-protocol-state-snapshot": "",
+              "node-info": "",
+              "execution-state-wal-chunks": ""
+            }
+          },
+          "flow-go-tag": "v0.10.1",
+          "flow-dps-tag": "",
+          "cadence-tag": "v0.9.2",
+          "docker-tag": "v0.10.1",
+          "access-node": "access-001.mainnet1.nodes.onflow.org:9000"
+        },
+        {
```

Comment:
haha, that's interesting interpretation, 0.9.

Reply:
Yeah, if we went with full numerics in the candidates then the other full Mainnet IDs would duplicate these and so numeric searches would have issues